evennia/utils/tests/test_containers.py

```diff
@@ -63,19 +92,45 @@ def test_start_with_typeclassed_script_loads_it(self):
     self.assertEqual(gsc.typeclass_storage['script_name'], GoodScript)
 
   @override_settings(GLOBAL_SCRIPTS={'script_name': {'typeclass': 'evennia.utils.tests.test_containers.BadScript'}})
-  def test_start_with_bad_typeclassed_script_skips_it(self):
+  def test_start_with_bad_typeclassed_script_raises(self):
+    gsc = containers.GlobalScriptContainer()
+
+    with self.assertRaises(Exception) as cm:
+        gsc.start()
+        self.fail("An exception was expected but it didn't occur.")
+
+    self.assertEqual(len(gsc.typeclass_storage), 1)
+    self.assertIn('script_name', gsc.typeclass_storage)
+
+  @override_settings(GLOBAL_SCRIPTS={'script_name': {'typeclass': 'evennia.utils.tests.test_containers.QuestionableScript'}})
+  def test_start_with_questionably_subclassed_script_skips_it(self):
+    # Make sure _BASE_SCRIPT_TYPECLASS is NOT DefaultScript but it's a subclass
+    assert issubclass(_BASE_SCRIPT_TYPECLASS, DefaultScript) and _BASE_SCRIPT_TYPECLASS != DefaultScript
+
     gsc = containers.GlobalScriptContainer()
 
     gsc.start()
 
-    self.assertEqual(len(gsc.typeclass_storage), 0)
-    self.assertNotIn('script_name', gsc.typeclass_storage)
+    self.assertEqual(len(gsc.typeclass_storage), 1)
```

Comment:
From not being a "loadable" script to a "loadable". If we had relaxed the `assert` instead, this new expectation will hold (a script subclassing `DefaultScript` is "loadable").